assets/resources/badusb/Install_qFlipper_macOS.txt

```diff
@@ -0,0 +1,34 @@
+ID 05ac:021e Apple:Keyboard
+REM Keep these 3 lines IF (and only if) it's the first time you are performing a badKB attack against a specific macOS target. 
+REM In fact, it helps Flipper Zero bypass the macOS keyboard setup assistant. Otherwise the attack will not start.
+
+REM Author: 47LeCoste
+REM Target: macOS
+DELAY 300
+GUI SPACE
+DELAY 1000
+STRING Terminal
+DELAY 1000
+ENTER
+DELAY 1000
+STRING curl -o ~/Downloads/qFlipper.dmg -L https://update.flipperzero.one/builds/qFlipper/1.3.2/qFlipper-1.3.2.dmg
```

Comment:
https://update.flipperzero.one/qFlipper/release/macos-amd64/dmg

Reply:
yep!!

Reply:
did it!